docs/content/docs/build/experimental-features.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Experimental Features
+aliases:
+  - "/docs/build/experimental-features/"
+author: scottn12
+---
+
+## Overview
+
+At any given time, there are experimental features within FluidFramework that are still being actively developed/tested. In order to allow developers to adopt and test these features in a controlled environment, we introduced experimental feature gates to enable these features and easily disable them if they cause unexpected behavior.
+
+### Proceed with Caution
+
+Experimental features are by definition not fully supported and **should not be used in production scenarios**. Moreover, experimental features may be **added, modified, or removed without warning** in any minor or major release.
+
+## API
+
+To enable experimental features, you will be required to provide an implementation of `IConfigProviderBase`. Please note the following about `IConfigProviderBase`:
```

Comment:
Assuming the API docs for `IConfigProviderBase` have been published, it might be nice to link to them here.

Reply:
Looks like it is not published unfortunately. Thoughts on alternatives?

Reply:
Should be published in telemetry-utils. We publish API docs updates on every change to `main`.

Reply:
https://fluidframework.com/docs/apis/telemetry-utils/iconfigproviderbase-interface

Reply:
But we'll want to link via the `apiref` or `relref` shortcode.